nativelib/src/main/scala/scala/scalanative/runtime/NativeThread.scala

```diff
@@ -72,15 +77,67 @@ trait NativeThread {
   }
 }
 
+private object ThreadStackSize {
+  final val Minimal = 64 * 1024
+  final val JVMDefault = 1024 * 1024 // 1MB is JVM default
+
+  // Additional stack size to compenstate memory for internals
+  private def extraThreadStackSize: Long = StackOverflowGuards.size
+
+  private val overrideDefaultThreadSize: Option[Long] = {
+    System.getenv("SCALANATIVE_THREAD_SIZE") match {
```

Comment:
`SCALANATIVE_THREAD_SIZE` is a too generic name. I think you should also mention the stack, like `SCALANATIVE_THREAD_STACK_SIZE`